app/views/practices/_marketplace_card.html.erb

```diff
@@ -29,7 +29,7 @@
           </h4>
         <%# end %>
         <div class="grid-row margin-bottom-1">
-          <p class="multiline-ellipses-1 line-height-sans-505 font-sans-sm practice-card-tagline"><%= practice.tagline %></p>
+          <p class="multiline-ellipses-1 line-height-sans-505 font-sans-sm practice-card-tagline"><%= practice.tagline.truncate(56) unless practice.tagline == nil%></p>
```

Comment:
can you do 
`practice.tagline&.truncate(56)`
instead? (get rid of the `unless`)

Reply:
Yeah - thats better way to handle nil.  Thx